tests/editor/editor.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * @license Copyright (c) 2003-2015, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+/* globals describe, it, expect, sinon, document */
+
+'use strict';
+
+var modules = bender.amd.require( 'editor' );
+
+describe( 'constructor', function() {
+	it( 'should create a new editor instance', function() {
+		var Editor = modules.editor;
+
+		var editor = new Editor( document.body );
```

Comment:
As soon as we'll start really initialising editors on this elements this will affect the other tests. I think we should use specific elements from the beginning, to avoid changing these tests in the future.